src/actionCreators/search.js

```diff
@@ -6,7 +6,9 @@ import { findAuthorityConfig } from 'utilities/authorityConfig'
 
 // eslint-disable-next-line max-params
 export const fetchSinopiaSearchResults = (query, queryFrom, resultsPerPage, sortField, sortOrder) => dispatch => getSearchResults(
-  query, queryFrom, resultsPerPage, sortField, sortOrder,
+  query, {
+    queryFrom, resultsPerPage, sortField, sortOrder,
```

Comment:
looks like this is sticking `resultsPerPage` in a field of the same name, but `getSearchResults` expects a field in the `options` hash called `size`?